internal/sendconfig/common_workflows.go

```diff
@@ -39,13 +41,16 @@ func UpdateKongAdminSimple(ctx context.Context,
 	enableReverseSync bool,
 	diagnostic util.ConfigDumpDiagnostic,
 	proxyRequestTimeout time.Duration,
+	promMetrics *metrics.CtrlFuncMetrics,
 ) ([]byte, error) {
 	// build the kongstate object from the Kubernetes objects in the storer
 	storer := store.New(*cache, ingressClassName, false, false, false, deprecatedLogger)
 	kongstate, err := parser.Build(deprecatedLogger, storer)
 	if err != nil {
+		promMetrics.ParseCounter.With(prometheus.Labels{"success": string(metrics.SuccessFalse)}).Inc()
```

Comment:
Please extract all string literals for metric labels (`"success"`, `"type"`) into reusable `consts`, so that we don't develop an unintentional discrepancy between field names in the future.

Reply:
The arguments could be better readability or better maintainability. I don't have strong preference, adjust according to your comments.